ansible/kubernetes.yml

```diff
@@ -1,4 +1,8 @@
 ---
 # This assumes you ran "make download" to download the kubespray files
-# 
+#
 - import_playbook: roles-external/kubespray/cluster.yml
+
+- hosts: etcd
```

Comment:
By default, `etcd` is not part of the hosts, right?

Reply:
https://github.com/wireapp/wire-server-deploy/blob/master/ansible/hosts.example.ini#L83

it is part by default, see above. Kubespray needs this to be in the hosts file.

Reply:
🤦‍♂ I looked in the file and missed it. Move along. Thanks!